### DISCUSSION TITLE
blueprintjs.com or tailwind css or ??? and Nextjs SSR

### DISCUSSION THREAD

**Discussion by thierryc** [vercel/next.js#13365] [Help]
I am trying to find the right css/js framework. 
SSR is important. Who has some advice? Any feedback is welcome. 

TIA

**Comment by ghost** [comment]
A CSS framework shouldn't cause any problem with SSR. I use tailwind with purge css and it works pretty nice. 
https://github.com/denoland/deno_website2 is a good example of next + tailwind.

**Comment by webdeb** [comment]
@thierryc if you have your own CI / Design. I would go with tailwindcss since its really a **DesignSystemFramework** instead of a **Components Library**.

If you are satisfied with ready to use Components like Material-UI / Blueprint / Bootstrap. Just go with them.

However, I've never worked on a project where a Components library covered 100% of my needs. And because I mostly had to learn the particular framework, the props, behaviour, the caveats etc. I've spended a lot of time. And it's actually a useless timeloss in the long run.

But with tailwind its different, because its basically css as classes, but simpler to write IMHO

**Comment by thierryc** [comment]
Thank you for your feedback. @searchableguy @webdeb
